impress-org/givewp
# Fix: set default email recipient to admin email

Resolves #4411

## Description


We can separate #4411 into two parts:
- Admin can save the empty emails.
   **I updated email recipient validation logic to save website default admin email as the recipient if admin saves empty email as recipient.**
- Admin is not able to add recipient after saving empty email receipt.
   **I added a migration that fix the broken admin email recipient setting value. Website default admin email will be added to these settings.**

## Testing Instructions


You should not able to reproduce issues with the pull request. Switch to `develop` Github branch to reproduce the issue and then test with the issue with this pull request. 

## Pre-review Checklist
